packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts

```diff
@@ -75,10 +75,7 @@ export class EdgelessShapeToolButton extends EdgelessToolButton<
     }
 
     .container-clip {
-      padding: 10000px 10000px 0;
```

Comment:
Leds to some unexpected effects #7139

Reply:
A very large layer will be created here.

Reply:
Yes, that's expected, to achieve the effect of `overflow: clip` only on the bottom edge. Is there any performance issue or other problem with this approach?

Reply:
The layer is always there. 

You can consider trying canvas overlayer, also create a separate element outside(fixed/absolute).

Reply:
<img width="1121" alt="Screenshot 2024-05-26 at 12 45 37" src="https://github.com/toeverything/blocksuite/assets/27926/35a9dbd7-9df3-45ff-beac-f553d2be3754">

Reply:
> Yes, that's expected, to achieve the effect of `overflow: clip` only on the bottom edge

clip does not work on padding